_activities/pyhep.md

```diff
@@ -0,0 +1,45 @@
+---
+title: PyHEP - "Python in HEP" Series of Workshops
```

Comment:
I'd also cut the "title" down to just "PyHEP Workshops". The current one is really too long for the drop down menu.

Reply:
I did not know the title serves in that menu.
In the meantime I got ruby/docker/jekyll... installed on my Ubuntu VM and could test things. Seems fine.